internal/uvm/create_lcow.go

```diff
@@ -168,6 +189,13 @@ func CreateLCOW(opts *OptionsLCOW) (_ *UtilityVM, err error) {
 		},
 	}
 
+	if *opts.UseGuestConnection {
```

Comment:
You could just write this as `if opts.UseGuestConnection == nil || *opts.UseGuestConnection` and you wouldn't need the fixup above.

Reply:
I liked the organization of having a section at the top to set defaults on options, and then use those values later knowing that they've all been set up properly. If you prefer it this way, I can change it, though.

Reply:
No I am fine with that. I will likely remove the pointers part later anyways for simplicity